notebook/starter_notebook.ipynb

```diff
@@ -0,0 +1,981 @@
+       "      <td>1.1375</td>\n",
+       "      <td>0.8750</td>\n",
+       "      <td>0.2875</td>\n",
+       "      <td>14.968536</td>\n",
+       "      <td>5.953395</td>\n",
+       "      <td>2.962523</td>\n",
+       "      <td>3.713785</td>\n",
+       "      <td>8.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  Sex  Length  Diameter  Height     Weight  Shucked Weight  Viscera Weight  \\\n",
+       "0   M  1.5750    1.2250  0.3750  31.226974       12.303683        6.321938   \n",
+       "1   I  1.2375    1.0000  0.3750  21.885814        7.654365        3.798833   \n",
+       "2   F  1.4500    1.1625  0.4125  28.250277       11.127179        7.016501   \n",
+       "3   I  1.3500    1.0250  0.3750  21.588144        9.738053        4.110678   \n",
+       "4   I  1.1375    0.8750  0.2875  14.968536        5.953395        2.962523   \n",
+       "\n",
+       "   Shell Weight   Age  \n",
+       "0      9.638830  10.0  \n",
+       "1      7.654365  19.0  \n",
+       "2      7.257472  11.0  \n",
+       "3      6.378637   9.0  \n",
+       "4      3.713785   8.0  "
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data = data.drop(['id'],axis=1)\n",
+    "data.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Baseline Model\n",
+    "\n",
+    "Let's start with the very basic model! We will going to be use linear regression model from scikit-learn.\\\n",
+    "This will be our very base model and your goal is to build a model that performs better!\\\n",
+    "\n",
+    "## Linear Regression\n",
+    "\n",
+    "It is a very fundamental algorithm in statistic where the simplest way to explain is to find the 'line of best fit'.\\\n",
+    "There will be different given factors(which we call them 'variables') such as length, diameter, etc. and the model will determine the age based on the given factors\\\n",
+    "As a data scientist, one of our main jobs is to find the relevant variables out of given information."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So first, we will start by importing the pre-built model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.linear_model import LinearRegression\n",
+    "from sklearn.model_selection import train_test_split"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since Linear Regression is technically a mathmetical equation(or prompt) so we will drop column 'sex' which is a string value, not integer or simply, number.\\\n",
+    "(Although sex might be, or probably, relavant information that we want to utilize but we will teach you how to utilize them later on)."
```

Comment:
(However, sex is probably relevant information that we will want to utilize which we will learn later on).